configs/next_router_vuejs.json

```diff
@@ -5,13 +5,19 @@
       "url": "https://next.router.vuejs.org/guide/",
       "selectors_key": "guide",
       "page_rank": 10,
-      "tags": ["guide"]
+      "tags": ["guide"],
+      "extra_attributes": {
+        "language": ["zh"]
+      }
     },
     {
       "url": "https://next.router.vuejs.org/api/",
       "selectors_key": "api",
       "page_rank": 8,
-      "tags": ["api"]
+      "tags": ["api"],
+      "extra_attributes": {
+        "language": ["zh"]
+      }
     }
```

Comment:
The crawler won't get the `.org/zh/x` endpoint here, you might need to go with something like: https://github.com/algolia/docsearch-configs/pull/3942/commits/04f65854de94d0e422e34b6f3b8cc75e246ec961

but we shouldn't merge the PR before https://github.com/algolia/docsearch-configs/pull/3942#issuecomment-815963322 is fixed

Reply:
that looks good, let me push the change

Reply:
Oh nvm, I somehow didn't see you pushed it already! 👍

Reply:
Right now, docs will always have the `en-US` in the lang attribute on the HTML element. Wouldn't that just always show English search results until I fix the lang attribute on my side?

Otherwise, yeah, I will have to push a fix first and deploy the docs. I imagine the search will be broken until this PR is merged. If that's the case what time would be the best to sync with you?

Reply:
Yes, as of now everything will be under the `en-US` lang so it will show the results for both languages under the `en-US` filter.

I'm on Paris timezone so anything between 9am - 1am is good for me, the crawl will be queued right after the merge

Reply:
oh perfect, same for timezone for me! Then it sounds like it will behave the same as right now so it should be good

Reply:
Sorry, what I meant was, right now the `zh` results are not crawled at all, after the merge, they will be but under the `en-US` lang.

In the meantime, we can still merge the fix for the `lang` selector so you'll be able to use the `tags`

Reply:
Sounds good to me. I guess we could also add a stop_url at `/zh` until I fix the problem of my end so it doesn't crawl the chinese docs under the same facet filter as the english ones

Reply:
Sure, I'm not on a computer right now so feel free to push and I'll merge!